airflow/task_runner/base_task_runner.py

```diff
@@ -119,7 +120,7 @@ def run_command(self, run_with, join_args=False):
         # Start daemon thread to read subprocess logging output
         log_reader = threading.Thread(
             target=self._read_task_logs,
-            args=(proc.stdout,),
```

Comment:
If this is the error I think it is it can be changed by this diff which is a bit better (I actually had a fix ready as part of another PR I am working on):
read_task_logs:
```
-            line = stream.readline()
+            line = stream.readline().encode('utf-8')
```